public/js/whydPlayer.js

```diff
@@ -496,19 +496,31 @@ function WhydPlayer() {
   // handlers for events coming from Playem
 
   var playemEventHandlers = {
-    onError: function(e) {
+    onError: function (e) {
       if (currentTrack.metadata.logData) logTrackPlay();
 
+      // will try to fallback failing tracks to deezer player
+      //Detect user agent for electron specific message
+      var userAgent = navigator.userAgent.toLowerCase();
+      var isElectronApp = false;
+      var failedTrackMessage;
+      if (userAgent.indexOf(' electron/') > -1) {
```

Comment:
Can we use the full `openwhyd-electron` name, to make the condition even more specific to our own desktop client?

```suggestion
      if (userAgent.indexOf(' openwhyd-electron/') > -1) {
```

Reply:
Yeah I think that works if the desktop client for openwhyd does have 'openwhyd-electron' in the user agent. I just used the code mentioned in the stackoverflow post that would work for any electron app I suppose